cfde_submit/client.py

```diff
@@ -130,12 +131,16 @@ def __init__(self, service_instance="prod", tokens=None):
                     be called instead.
                     **Default**: ``None``.
         """
-        self.service_instance = service_instance
+        self.__service_instance = os.getenv("CFDE_SUBMIT_SERVICE_INSTANCE", "prod")
         self.__remote_config = {}  # managed by property
         self.__tokens = {}
         self.__flow_client = None
+        self.local_config = fair_research_login.ConfigParserTokenStorage(
+            filename=self.config_filename
+        )
         self.__native_client = fair_research_login.NativeClient(client_id=self.client_id,
-                                                                app_name=self.app_name)
+                                                                app_name=self.app_name,
+                                                                token_storage=self.local_config)
```

Comment:
Changing the storage location wasn't really in-scope for these changes, but I really think we should start using a custom location to store cfde-submit related data.

Reply:
I tested these changes on midway and was able to run successfully. Everything looks good to me.

Reply:
Great! You might also try testing this in a local environment. If you run into any problems with auth, for example, someone else out there will likely encounter the same issue at some point.

Please merge when you're ready.